tests/snuba/api/endpoints/test_organization_events_mep.py

```diff
@@ -1904,11 +1904,3 @@ def test_custom_measurement_size_filtering(self):
     @pytest.mark.xfail(reason="Having not supported")
     def test_having_condition(self):
         super().test_having_condition()
-
-    @pytest.mark.xfail(reason="Metrics layer failing to support ordering by apdex")
```

Comment:
can we keep these tests to demonstrate of work? instead of skip

Reply:
@wmak told me to remove those.

Reply:
gotcha